include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+#ifndef _NBL_VIDEO_S_PHYSICAL_DEVICE_LIMITS_H_INCLUDED_
+#define _NBL_VIDEO_S_PHYSICAL_DEVICE_LIMITS_H_INCLUDED_
+
+#include <type_traits>
+#include "nbl/asset/utils/IGLSLCompiler.h" // asset::IGLSLCompiler::E_SPIRV_VERSION
+
+
+namespace nbl::video
+{
+
+struct SPhysicalDeviceLimits
```

Comment:
seems we might be missing a few things:
<html><body>
<!--StartFragment-->

framebufferColorSampleCounts | Limit | Listing
-- | -- | --
framebufferDepthSampleCounts | Limit | Listing
framebufferNoAttachmentsSampleCounts | Limit | Listing
framebufferStencilSampleCounts | Limit | Listing

<!--EndFragment-->
</body>
</html>framebufferColorSampleCounts	Limit	[Listing](https://vulkan.gpuinfo.org/displaydevicelimit.php?name=framebufferColorSampleCounts&platform=all)
framebufferDepthSampleCounts	Limit	[Listing](https://vulkan.gpuinfo.org/displaydevicelimit.php?name=framebufferDepthSampleCounts&platform=all)
framebufferNoAttachmentsSampleCounts	Limit	[Listing](https://vulkan.gpuinfo.org/displaydevicelimit.php?name=framebufferNoAttachmentsSampleCounts&platform=all)
framebufferStencilSampleCounts	Limit	[Listing](https://vulkan.gpuinfo.org/displaydevicelimit.php?name=framebufferStencilSampleCounts&platform=all)